src/index.js

```diff
@@ -0,0 +1,59 @@
+const isTest = process.env.NODE_ENV === 'test';
+
```

Comment:
we also might want to have some sort of generic module that checks for `src` at the project root?